src/main/java/com/endava/soa_5ed/task3_oop/Cat.java

```diff
@@ -0,0 +1,84 @@
+package com.endava.soa_5ed.task3_oop;
+
+public class Cat extends Feline implements Tail {
+    private double size;
+    private String name;
+
+    public Cat(String gender, String furColor, String name) {
+        super(gender, furColor);
+        this.name = name;
+        System.out.println("############ Creating Cat ############");
+    }
+
+    public Cat(String gender, String furColor, double size, String name) {
+        super(gender, furColor);
+        this.size = size;
+        this.name = name;
+        System.out.println("############ Creating Cat ############");
```

Comment:
And instead sout , you may use log.info(<your message here>). 
In production, mostly, we are using logs instead of souts.

Reply:
done